### DISCUSSION TITLE
Complejidad en TP Integrador

### DISCUSSION THREAD

**Discussion by martinjguevara** [INGCOM-UNRN-P1/cursada-2023#232] [Preguntas y respuestas] [answered]
Hay que incluir un archivo markdown con el calculo de la complejidad de las funciones del programa? La consigna no indica nada respecto de la complejidad

**Comment by martinvilu** [comment, marked as answer]
Es una buena pregunta para la defensa, alguna función debe de ser ‘núcleo’ al programa, fíjate de estudiarla.
